packages/slice-machine/components/AppLayout/Navigation/Menu/Desktop.tsx

```diff
@@ -125,7 +125,7 @@ const Desktop: React.FunctionComponent<{ links: LinkProps[] }> = ({
                   latest: latestVersion && latestVersion.versionNumber,
                   latestNonBreaking: changelog.latestNonBreakingVersion,
                 });
-                return router.push("/changelog");
+                void router.push("/changelog");
```

Comment:
why void?

Reply:
that's a linter issue, the function returns a promise when the return type shouldn't be.

Knowing that returning the promise doesn't change anything, I don't return anymore but I use void to tell my code that it doesn't return anything.

Reply:
ok